src/listeners/log-to-console.js

```diff
@@ -53,10 +53,9 @@ export let logToConsole = function(cfg = {}) {
       return;
     }
 
-    let maxLevelName = cfg.level;
-    let maxLevel = logger.levels[maxLevelName];
-    maxLevel = _.floor(maxLevel / 10) * 10 + 10 - 1; // round up to next level, not inclusive
-    if (entry._level > maxLevel) {
+    let maxLevelCode = logger.levelToLevelCode(cfg.level);
+    maxLevelCode = _.floor(maxLevelCode / 10) * 10 + 10 - 1; // round up to next level, not inclusive
```

Comment:
why not `(_.floor(maxLevelCode / 10) + 1) * 10 - 1;` 
or just™ `_.floor(maxLevelCode / 10) * 10 + 9`?

Reply:
> or _.ceil(maxLevelCode / 10) * 10 - 1 maybe

won't work for an already rounded level like `60`, which will become 59 as a result

Reply:
to be continued in https://github.com/tobiipro/minlog/pull/8